Xamarin.Forms.Platform.Android/Renderers/LabelRenderer.cs

```diff
@@ -23,6 +23,8 @@ public class LabelRenderer : ViewRenderer<Label, TextView>
 		FormsTextView _view;
 		bool _wasFormatted;
 
+		readonly MotionEventHelper _motionEventHelper = new MotionEventHelper();
```

Comment:
The only thing which holds a reference to MotionEventHelper is the renderer; when the renderer is collected, MotionEventHelper goes away, too. Setting it to null during dispose doesn't make any difference.